app/src/org/commcare/dalvik/activities/LoginActivity.java

```diff
@@ -566,4 +586,61 @@ public CustomProgressDialog generateProgressDialog(int taskId) {
     public boolean isBackEnabled() {
         return false;
     }
+    
+    private void refreshView() {
+        Spinner spinner = (Spinner) findViewById(R.id.app_selection_spinner);

```

Comment:
You can declare these as fields in the activity which will prevent the need for this.